lib/Echidna/Campaign.hs

```diff
@@ -46,6 +47,8 @@ data CampaignConf = CampaignConf { testLimit     :: Int
                                  , knownCoverage :: Maybe (Map W256 (Set Int))
                                    -- ^ If applicable, initially known coverage. If this is 'Nothing',
                                    -- Echidna won't collect coverage information (and will go faster)
+                                 , timeout       :: Integer
+                                   -- ^ Timeout in picoseconds
```

Comment:
I don't think timeout should be in picoseconds. milliseconds or seconds seems more normal. Granted, the config option is seconds but I think that it should be reflected in `CampaignConf`.